scripts/just.config.ts

```diff
@@ -74,6 +75,7 @@ export function preset() {
   task('storybook:start', startStorybookTask());
   task('storybook:build', buildStorybookTask());
   task('babel:postprocess', babel);
+  task('token-pipeline', tokenPipeline);
```

Comment:
its not necessary to register this into just IMO, can be run directly via node. WDYT?

Reply:
Something like
```
  "scripts": {
    "token-pipeline": "node -r ../../scripts/ts-node-register ../../scripts/tasks/token-pipeline.ts"
  },
```
in `react-theme/package.json`?

Reply:
yup that could work